Roulette.js

```diff
@@ -34,3 +56,38 @@ module.exports = class Roulette {
 		});
 	}
 };
+
+function generateSpaces() {
```

Comment:
Thought I didn't need new ones for each instance of the class. Will this not work the way I intended it to?